README.md

```diff
@@ -45,7 +45,7 @@ so end-users have full guidance in using this tool-set and do not have to search
 
 ## Getting Started
 
-First, start a Jenkins instance with JCasC [plugin](https://plugins.jenkins.io/configuration-as-code) installed.
+First, start a Jenkins instance with the [configuration-as-code](https://plugins.jenkins.io/configuration-as-code) plugin installed.
```

Comment:
I often reference the plugin by JCasC since configuration-as-code plugin does not sit well with others especially when you bring it up in talks, meeting etc. So perhaps this is not a good change :sweat_smile: or perhaps add a section about `also known as JCasC (Jenkins Configuration as Code)` or better yet `better known as JCasC (Jenkins Configuration as Code)`

Reply:
I'll have a look ait it again. I already tried to make it more consistent but couldn't decide on exact 1 naming. Unfortunately there are many ways how it is named "JCasC", "Configuration-as-code", "configuration-as-code", "Configuration as Code", ... Sometimes "configuration-as-code" should be used because at the same time 2nd plugin "configuraiton-as-code-support" is also referred to.

Reply:
I added another commit with more improvements.
It also changes a lot of `/configuration.?as.?code/i` occurrences with "JCasC".